### DISCUSSION TITLE
[persist] custom Localstorage with expiry is not work

### DISCUSSION THREAD

**Discussion by vaynevayne** [pmndrs/zustand#1879] [Bug report]
https://codesandbox.io/s/zustand-state-with-url-hash-demo-forked-5q6gfq?file=/src/store/index.ts

Set the expiration time to 2 milliseconds, but get count after 2ms, the same value as before

**Comment by dai-shi** [comment]
It seems working as I expect.


https://github.com/pmndrs/zustand/assets/490574/ab0663cf-74b6-475a-b57d-97c9fd6d4850

**Reply by vaynevayne** [reply]
expires is set to 2 milliseconds,it should be remove

**Reply by dai-shi** [reply]
Isn't it 2 seconds?

**Reply by vaynevayne** [reply]
Pretty much. Anyway, he's still here. The expiration hasn't kicked in

You also want zustand middleware to support this feature naturally
